### ISSUE TITLE
Minor cleanup: README

### ISSUE THREAD

**Issue by luke-hill** [cucumber/godog#221]
This area of the readme needs some rework, W3C spec is now conformant on most browsers and we should be recommending people use selenium - Just point them to the drivers and use proper browsers. https://github.com/cucumber/godog#testing-browser-interactions

I know a few people said they wanted to help maintain, this would be a good first issue.

**Comment by lonnblad** [comment]
Hi

I haven't used Selenium for 2 years now and at my office, we instead use Cypress for this verification, adding Gherkin only as comments to our TCs.

Is there a good support for Gherkin in Selenium or are there any other reasons why we should promote Selenium over other tools?

**Comment by jaysonesmith** [comment]
We use Cypress as well. @luke-hill Do any of the other official implementations specifically point folks to Selenium?

**Comment by luke-hill** [comment]
Selenium is the w3c standard and works within it, others I'm not so sure about.

Aside from JS, most languages wrap selenium, JS is the odd one out in that everyone seems to want to replace it. So try avoiding JS specific situations and just recommend it, or point them to w3c

Atm the only time we reference automation frameworks is capybara using selenium / rack test inside cucumber-rails / ruby otherwise we don't have refs.

TL;DR - When all else fails, point to w3c, or something which works cross language like selenium

**Comment by inluxc** [comment]
I use godog with [tebeka](https://github.com/tebeka/selenium) for Selenium Tests and they work great together.

**Comment by lonnblad** [comment]
@inluxc interesting, looks nice! :)

The main reason we are using godog and cypress is due to that the backend developers work in go ergo godog and frontend developers work in js ergo cypress, so that they have an easier time.

**Comment by aslakhellesoy** [comment]
Unless our code has specific integration with a particular browser automation tool, I don't think we should be making any recommendations about what to use or not use.

There are so many different contexts and constraints that we cannot make a general recommendation IMO.

**Comment by luke-hill** [comment]
Given this issue I wrote ages ago and re-looking at it. I think it was specifically about the godog readme file. So yeh I think the OP still stands. We should try remove any notes about stuff which is actively unmaintained.

@inluxc does the tool you mention have a direct integration with cucumber? I don't use go so I don't know, but I think also what aslak mentions about avoiding generalisations

**Comment by lonnblad** [comment]
Hi, thanks for the comments.

What I take from this is that we should remove this section from the README
https://github.com/cucumber/godog#testing-browser-interactions

**Comment by luke-hill** [comment]
Pretty much yes. Or maybe update it to something relevant / supported.
